crates/cast/bin/cmd/call.rs

```diff
@@ -225,7 +226,7 @@ impl CallArgs {
             }
         };
 
-        req.set_input(data.into());
+        req.set_input::<Bytes>(data.into());
```

Comment:
I don't think the type hint is required
```suggestion
        req.set_input(data);
```